README.md

```diff
@@ -43,6 +43,11 @@ Create a For Loop that outputs the following string for each piece of fruit in t
 
 ```js
 // write code here
+for (var i = 0; i < 1; i++) {
+  for (var j = 0; j < 3; j++) {
+    console.log(foods[i][j]);
+  }
+}
 ```
```

Comment:
This works, although it would be better to replace `1` and `3` with values representing the length of the arrays in question (i.e., `foods.length` and, in this case, `i.length`).